### PR TITLE
Standardize systemctl timers

### PR DESCRIPTION
**Pull request by ecdye**
Signed-off-by: Ethan Dye <mrtops03@gmail.com>

### CODE REVIEW

File: functions/backup.bash [hunk 1]
```diff
@@ -389,16 +389,16 @@ mirror_SD() {
     echo "Taking a raw partition copy, be prepared this may take long such as 20-30 minutes for a 16 GB SD card"
     if ! cond_redirect dd if="${src}p1" bs=1M of="${dest}1" status=progress; then echo "FAILED (raw device copy of ${dest}1)"; dirty="yes"; fi
     if ! cond_redirect dd if="${src}p2" bs=1M of="${dest}2" status=progress; then echo "FAILED (raw device copy of ${dest}2)"; dirty="yes"; fi
-    origPartUUID=$(blkid "${src}p2" | sed -n 's|^.*PARTUUID="\(\S\+\)".*|\1|p' | sed -e 's/-02//g')
-    if ! partUUID=$(yes | cond_redirect set-partuuid "${dest}2" random | awk '/^PARTUUID/ { print substr($7,1,length($7) - 3) }'); then echo "FAILED (set random PARTUUID)"; dirty="yes"; fi
+    origPartUUID="$(blkid "${src}p2" | sed -n 's|^.*PARTUUID="\(\S\+\)".*|\1|p' | sed -e 's/-02//g')"
```
Comment (mstormi) [inline functions/backup.bash:392]: Tested ? Some shellcheck recommendations affect functioning

File: includes/amandaBackupDB.service-template [hunk 2]
```diff
@@ -1,6 +1,7 @@
 [Unit]
-Description=nightly backup: Amanda config, DB, logs
+Description=Make nightly backup of Amanda database
 After=network.target network-online.target
+Wants=amandaBackupDB.timer
```
Comment (mstormi) [inline includes/amandaBackupDB.service-template:5]: tested ? useful ? Haven't seen those before

File: includes/amandaBackupDB.timer [hunk 3]
```diff
@@ -1,7 +1,9 @@
 [Unit]
-Description=Make backup of Amanda database
+Description=Make nightly backup of Amanda database
+Requires=amandaBackupDB.service
```
Comment (mstormi) [inline includes/amandaBackupDB.timer:3]: same here: useful? tested ?

File: functions/backup.bash (hunk 1 above)
Comment (ecdye) [reply functions/backup.bash:392]: Yes, that was just a simple double quote.

File: includes/amandaBackupDB.service-template (hunk 2 above)
Comment (ecdye) [reply includes/amandaBackupDB.service-template:5]: Yes just a safer and more specific way of referring to timers

**Review by mstormi — APPROVED** [review]

**Pull request merged**